src/Microsoft.AspNet.SignalR.Core/Infrastructure/MemoryPoolTextWriter.cs

```diff
@@ -3,10 +3,11 @@
 using System;
 using System.Text;
 using System.IO;
+using System.Globalization;
 
 namespace Microsoft.AspNet.SignalR.Infrastructure
 {
-    public class MemoryPoolTextWriter : TextWriter
+    internal class MemoryPoolTextWriter : TextWriter
```

Comment:
I don't think there is a specific reason that so many of SignalR's seemingly internal types are public. @davidfowl (at least at the time) had the philosophy to make all types public unless there was a specific compelling reason not to.

At no point did we ever expect SignalR users to have to include `using  Microsoft.AspNet.SignalR.Infrastructure` in their code. Hopefully making this internal won't break anyone. I doubt it will.